lib/matplotlib/axes/_base.py

```diff
@@ -2810,6 +2810,19 @@ def get_xlim(self):
         """
         return tuple(self.viewLim.intervalx)
 
+    def _validate_axis_limits(self, limit, convert):
+        """
+        Raise ValueError if specified axis limits are infinite.
+
```

Comment:
Name is good. A slight tweak to that docstring,
"Returns : The limit value after call to convert(), or None if limit is None."